spec/requests/create_collection_spec.rb

```diff
@@ -193,9 +193,9 @@
             collection = Collection.last
             expect(response).to have_http_status(:found)
             expect(response).to redirect_to(collection_path(collection))
-            expect(collection.managed_by.map(&:sunetid)).to eq ['maya.aguirre', 'jcairns']
-            expect(collection.reviewed_by.map(&:email)).to eq %w[maya.aguirre@stanford.edu
-                                                                 jcairns@stanford.edu faridz@stanford.edu]
+            expect(collection.managed_by.map(&:sunetid).sort).to eq ['jcairns', 'maya.aguirre']
```

Comment:
Rspec has built in function, `contain_exactly` that you can use for this case.

Reply:
changed